roles/storage_nfs_lvm/README.md

```diff
@@ -0,0 +1,104 @@
+# storage_nfs_lvm
+
+This role is useful to create and export nfs disks for openshift persistent volumes.
+It does so by creating lvm partitions on an already setup pv/vg, creating xfs 
+filesystem on each partition, mounting the partitions, exporting the mounts via NFS
+and creating a json file for each mount that an openshift master can use to
+create persistent volumes.
+
+## Requirements
+
+* NFS server with NFS, iptables, and everything setup.
+
+* A lvm volume group called openshift3vg created on the nfs server
```

Comment:
we're trying really hard not to put 3 in our names as this will likely be our foundation for v4 and on as well. 

Consider changing the VG to just `openshiftvg`